ui/scss/component/_main.scss

```diff
@@ -102,7 +102,9 @@
 }
 
 .main__document-wrapper {
-  width: 40em;
+  max-width: 100%;
+  min-width: 80%;
+  width: fit-content;
```

Comment:
I don't think this should change. If it needs to be wider for non-markdown files we can do that, but for the most part, text files should have a static width like `40em`

Reply:
https://github.com/lbryio/lbry-desktop/pull/3511/commits/eeaa550a3715a96c47d638b58d269dc492893886